x/concentrated-liquidity/lp.go

```diff
@@ -171,7 +171,7 @@ func (k Keeper) WithdrawPosition(ctx sdk.Context, owner sdk.AccAddress, position
 		return sdk.Int{}, sdk.Int{}, types.NotPositionOwnerError{PositionId: positionId, Address: owner.String()}
 	}
 
-	// Defense in depth, requestedLiquidityAmountToWithdraw should always be a positive value.
+	// Defense in depth, requestedLiquidityAmountToWithdraw should always be a value that is BGE than 0.
```

Comment:
```suggestion
	// Defense in depth, requestedLiquidityAmountToWithdraw should always be a value that is GE than 0.
```

(unless BGE is a term that I'm not familiar with)